src/posts/2020-09-29-introducing-WAI.md

```diff
@@ -0,0 +1,59 @@
+category: Background
+author: Hamsa Harcourt
+further_reading:
+  - title: About W3C WAI
+    url: https://www.w3.org/WAI/about/
+    source: W3C Web Accessibility Initiative
+  - title: "What is Web Accessibility Initiative (WAI)?"
+    url: https://www.washington.edu/doit/what-web-accessibility-initiative-wai
+    source: "DO-IT - Disabilities, Opportunities, Internetworking, and Technology"
+  - title: Web Accessibility Initiative
+    url: https://en.wikipedia.org/wiki/Web_Accessibility_Initiative
+    source: Wikipedia
+date: 2020-09-29
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+
+The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the lack of standard guidelines and the divergence of individual initiatives that provided no real solution to people, the **Web Accessibility Initiative**(WAI) was created.
+
+## Introduction
+
+WAI is one of the initiatives created by the [World Wide Web Consortium(W3C)](https://www.w3.org/Consortium/) as an effort to improve accessibility on the web.
+
+According to Tim Bernes-Lee, Director of the W3C and inventor of the World Wide Web:
+
+<blockquote>
+	<p>Worldwide, there are more than 750 million people with disabilities. As we move towards a highly connected world, it is critical that the web be usable by anyone, regardless of individual capabilities and disabilities.</p>
+</blockquote>
+
+WAI pursues accessibility through the following ways:
+
+- Ensuring that the web technology support accessibility.
+- Developing guidelines for web content and application, browsers ad authoring tools.
+- Developing tools to evaluate and facilitate accessibility.
+- Supporting education and outreach on web accessibility.
+- Co-ordinating with research and development that may impact the future of accessibility on the web.
+- Promoting harmonized international uptake of web accessibility standards.
+
+## Organization
+
+WAI consists of several [working groups](https://en.wikipedia.org/wiki/Working_group) and [special interest groups](https://en.wikipedia.org/wiki/Special_Interest_Group). Each working together for a common goal: improve accessibility on the web. The following are the working and special interest groups :
+
+- [Accessibility Guideline Working Group (formerly the Web Content Accessibility Guidelines Working Group)](https://www.w3.org/WAI/GL/).
+- [Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/).
+- [Accessible Rich Internet Applications (ARIA) Working Group](https://www.w3.org/WAI/ARIA/).
+- [Education and Outreach Working Group (EOWG)](https://www.w3.org/WAI/about/groups/eowg/).
+- [WAI Interest Group (WAI IG)](https://www.w3.org/WAI/about/groups/waiig/).

```

Comment:
Where did you source this list from?

Reply:
The list was sourced from their official site: https://www.w3.org/WAI/about/. I also provided the link in case of further reading